.gitlab-ci.yml

```diff
@@ -30,6 +30,8 @@ RPM:
     EXTRA_REPO_PATH_SEGMENT: "gitlab/"
   script:
     - sh "schutzbot/mockbuild.sh"
+  after_script:
+    - schutzbot/update_github_status.sh update
```

Comment:
This should be already included from `extends: .terraform` no? Or does it just override the after_script from that?

Reply:
it overrides the value from the `.terraform` leaving only this script to be executed and not all of the others.